modules/ROOT/pages/http/http-start-app-brows-task.adoc

```diff
@@ -1,22 +1,23 @@
-= To Start an App from a Browser
+= Start an App from a Browser
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In this procedure, you configure an HTTP Listener. You set an HTTP URL consisting of a host name, a port, and a path. To listen on a specific port for every network interface on the server, you set host to `0.0.0.0`. Using `localhost` listens for incoming requests generated inside the server. When the Listener hears the request in the browser, it initiates the app.
+In this procedure, you configure an HTTP Listener to specify a host name, a port, and a path. To listen on a specific port for every network interface on the server, set the host to `0.0.0.0`. Using `localhost` listens for incoming requests generated inside the server. Start a browser and submit a URL. The result appears in the Anypoint Studio Console.
 
-. Create a new project.
-. From Mule Palette, drag an HTTP Listener to a new flow.
+. In Studio, create a new project from *File* > *New* > *Mule Project*.
```

Comment:
You are not really creating a new project from *File* > *New* > *Mule Project*.  Maybe "create a new project by selecting *File* >..."